<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCUProg</class>
 <widget class="QWidget" name="MCUProg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MCU Programmer</string>
  </property>
  <layout class="QVBoxLayout" name="vLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hLayout1">
     <item>
      <widget class="QLabel" name="lblMCU">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>MCU ：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMCU">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblAddr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>起始：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAddr">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>大小：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSize">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>62</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout2">
     <item>
      <widget class="QLabel" name="lblDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>DLL ：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbDLL"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDLL">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout3">
     <item>
      <widget class="QLabel" name="lblHEX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>文件：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbHEX"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnHEX">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout4">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblMode">
       <property name="text">
        <string>模式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMode">
       <item>
        <property name="text">
         <string>ARM SWD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ARM JTAG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RISC-V JTAG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RISC-V cJTAG</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSpeed">
       <item>
        <property name="text">
         <string>1 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80 MHz</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="prgInfo">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>60</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnChipErase">
       <property name="text">
        <string>整片擦除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnErase">
       <property name="text">
        <string>擦除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWrite">
       <property name="text">
        <string>擦写</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
